pyproject.toml

```diff
@@ -15,36 +15,36 @@ meta-lookup = 'main:main'
 
 [tool.poetry.dependencies]
 python = "^3.9.1"
```

Comment:
I dropped all the patch versions and updated the lock file, so we will use the latest patch for all dependencies.